integrations/cypress/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@bigtest/cypress",
+  "version": "0.0.2",
+  "description": "Cypress Integration for BigTest Interactors",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/thefrontside/bigtest.git",
+  "author": "Frontside Engineering <engineering@frontside.com>",
+  "license": "MIT",
+  "files": [
+    "dist/**/*",
+    "src/**/*",
+    "README.md"
+  ],
+  "dependencies": {
+    "@bigtest/globals": "^0.7.3",
+    "@bigtest/interactor": "^0.23.0",
+    "cypress": "^5.6.0"
+  },
+  "scripts": {
+    "start": "cd ../../packages/todomvc/ && yarn && yarn prepack && yarn bigtest-todomvc 3000",
```

Comment:
is todomvc not redundant now that sample application got merged?

Reply:
I'm going to replace todomvc with the sample app in a separate pr.